CppCoreGuidelines.md

```diff
@@ -18842,7 +18842,7 @@ Source file rule summary:
 * [SF.4: Include `.h` files before other declarations in a file](#Rs-include-order)
 * [SF.5: A `.cpp` file must include the `.h` file(s) that defines its interface](#Rs-consistency)
 * [SF.6: Use `using namespace` directives for transition, for foundation libraries (such as `std`), or within a local scope (only)](#Rs-using)
-* [SF.7: Don't write `using namespace` at global scope in a header file](#Rs-using-directive)
+* [SF.7: Don't modify the global namespace in header files](#Rs-modify-global-namespace)
```

Comment:
I think this makes the name of this rule much less recognizable.
When I read "modify the global namespace" I think of adding new declarations to std, not using namespace std